test/onnx/test_fx_op_consistency.py

```diff
@@ -79,6 +416,64 @@
 # fmt: on
 
 SKIP_XFAIL_SUBTESTS: tuple[onnx_test_common.DecorateMeta, ...] = (
+    xfail(
+        "addmm",  # xfail can't only use dtypes to catch all cases
+        matcher=lambda sample: sample.input.dtype
+        in (torch.uint8, torch.int8, torch.int16),
+        reason=onnx_test_common.reason_onnx_script_does_not_support(
+            "Add", "int8, int16, uint8"
+        ),
+    ),
+    skip(
+        "all",
+        matcher=lambda sample: not (len(sample.kwargs) == 0),
+        reason="Need dispatcher: this Aten overload only support one tensor as input by design",
+    ),
+    skip(
+        "amax",
+        matcher=lambda sample: len(sample.input.shape) == 0,
+        reason="fixme (core dump): ORT aborts on scalar inputs to ReduceMax-18",
```

Comment:
I recall seeing this before but just to confirm. Does ort 1.15 still core dump?

Reply:
It raises error.